sdk/tvOS/OoyalaTVSkinSDK/OOTVGestureManager.m

```diff
@@ -89,4 +116,32 @@ - (void)togglePlay:(id)sender {
   [self.controller showProgressBar];
 }
 
+- (void)onSwipe:(id)sender {
+  if (sender == self.panGesture) {
+    CGPoint currentPoint = [self.panGesture translationInView:self.controller.view];
+    CGFloat seekScale = self.controller.player.duration / 1920.0;
```

Comment:
oh yea! that would be cool. Letting them choose from a range between a max limit (the view width) and a small limit, the lowest width we think it would look good, maybe a 1/3 of the view width or something like that.